Source/WebCore/accessibility/AXCoreObject.h

```diff
@@ -1412,8 +1412,8 @@ class AXCoreObject : public ThreadSafeRefCountedAndCanMakeThreadSafeWeakPtr<AXCo
 
 inline Vector<AXID> axIDs(const AXCoreObject::AccessibilityChildrenVector& objects)
 {
-    return objects.map([] (const auto& object) {
-        return object ? object->objectID() : AXID();
+    return WTF::compactMap(objects, [](const auto& object) {
```

Comment:
This is a behavior change. The old version would have nulls in the vector, I think, and the new one omits them. Is that OK?

Reply:
Call sites expects the IDs to be valid. Maybe I can use map() instead of compactMap() and assert that the IDs are valid for clarity.